scripts/augeas/nutupsdconf.aug.in

```diff
@@ -55,8 +55,14 @@ let upsd_certfile = [ opt_spc . key "CERTFILE" . sep_spc . store path . eol ]
  * ALLOW_NO_DEVICE Boolean
  * STATEPATH path
  * LISTEN interface port
- *    Multiple LISTEN addresses may be specified. The default is to bind to 0.0.0.0 if no LISTEN addresses are specified.
- *    LISTEN 127.0.0.1 LISTEN 192.168.50.1 LISTEN ::1 LISTEN 2001:0db8:1234:08d3:1319:8a2e:0370:7344
+ *    Multiple lines each with one LISTEN address (or host name) and an optional
+ *    port may be specified. The default is to bind to IPv4 and IPv6 "localhost"
+ *    addresses (subject to CLI options `-4` or `-6` constraining IP version or
+ *    OS kernel/configuration support), if no LISTEN addresses are specified.
```

Comment:
This points out: in the old code, if one did listen ::0 with -6, would nut have accepted v4 connections if v6only=0, and is this a bug, now fixed?  (Now we see the violence inherent in the sytsem -- another reason not to like mapped addresses.)

Reply:
I did not test specifically with a `-6`, but assume `LISTEN ::` could still handle IPv4. This is less of a concern for specific addresses/hostnames.

Reply:
I typed too fast.   What I meant is that if the user gives a v6 address literal, then if that socket could receive any v4 connections via mapped addresses, then that is a bug.   However, a strategy of always setting v6only=1 will avoid this.